.gitignore

```diff
@@ -0,0 +1,25 @@
+/Makefile
+/Makefile.in
+/aclocal.m4
+/autom4te.cache/
+/config/
```

Comment:
How lazy of you - ignoring the entire directory. :P I would prefer to list all the files put here instead.